src/main/java/com/gtnewhorizon/randomboubles/recipes/rings/InfusionRecipes.java

```diff
@@ -0,0 +1,427 @@
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 6), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), new ItemStack(BaubleItems.aquaRing, 1, 2),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 5), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 3) }));
+        RingResearch.recipeList.put(
+            "RBExoticRingOfTheSky",
+            ThaumcraftApi.addInfusionCraftingRecipe(
+                "RBEXOTICRINGOFTHESKY",
+                new ItemStack(BaubleItems.combinationRing, 1, 1),
+                5,
+                (new AspectList()).add(Aspect.AIR, 128)
+                    .add(Aspect.ORDER, 128)
+                    .add(Aspect.FIRE, 128)
+                    .add(Aspect.MAGIC, 128)
+                    .add(Aspect.AURA, 96)
+                    .add(Aspect.ENERGY, 64),
+                new ItemStack(ConfigItems.itemEldritchObject, 1, 3),
+                new ItemStack[] { new ItemStack(BaubleItems.combinationRing, 1, 0),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 0), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 4), new ItemStack(BaubleItems.combinationRing, 1, 0),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 6), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), new ItemStack(BaubleItems.combinationRing, 1, 0),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 4), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 0) }));
+        RingResearch.recipeList.put(
+            "RBExoticRingoftheEarth",
+            ThaumcraftApi.addInfusionCraftingRecipe(
+                "RBEXOTICRINGOFTHEEARTH",
+                new ItemStack(BaubleItems.combinationRing, 1, 4),
+                5,
+                (new AspectList()).add(Aspect.EARTH, 128)
+                    .add(Aspect.ENTROPY, 128)
+                    .add(Aspect.WATER, 128)
+                    .add(Aspect.MAGIC, 128)
+                    .add(Aspect.AURA, 96)
+                    .add(Aspect.ENERGY, 64),
+                new ItemStack(ConfigItems.itemEldritchObject, 1, 3),
+                new ItemStack[] { new ItemStack(BaubleItems.combinationRing, 1, 3),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 3), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 5), new ItemStack(BaubleItems.combinationRing, 1, 3),
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 6), salisMundus,
+                    new ItemStack(ConfigBlocks.blockCrystal, 1, 1), new ItemStack(BaubleItems.combinationRing, 1, 3),
```

Comment:
Fire instead of Water
```suggestion
                    new ItemStack(ConfigBlocks.blockCrystal, 1, 2), salisMundus,
                    new ItemStack(ConfigBlocks.blockCrystal, 1, 6), salisMundus,
                    new ItemStack(ConfigBlocks.blockCrystal, 1, 2), new ItemStack(BaubleItems.combinationRing, 1, 3),
```